Source/Scene/Scene.js

```diff
@@ -1214,30 +1219,30 @@ define([
             }
         }
 
+        if (scene._globeDepth.supported) {
+            scene._globeDepth.update(context);
+            scene._globeDepth.clear(context, passState, clearColor);
+        }
+
         var useOIT = !picking && renderTranslucentCommands && defined(scene._oit) && scene._oit.isSupported();
         if (useOIT) {
-            scene._oit.update(context);
+            scene._oit.update(context, scene._globeDepth.framebuffer);
```

Comment:
Are we using `_globeDepth.framebuffer` without an explicit check for `supported` here?  I guess it is implicit because OIT is supported, but is that really clear enough?